server/api.js

```diff
@@ -1,12 +1,54 @@
 import { Router } from "express";
-
+// import { Pool } from "pg";
+import db from "./db";
 import logger from "./utils/logger";
 
+
 const router = Router();
 
-router.get("/", (_, res) => {
-	logger.debug("Welcoming everyone...");
-	res.json({ message: "Hello, world!" });
+// router.get("/", (_, res) => {
+// 	logger.debug("Welcoming everyone...");
+// 	res.json({ message: "Hello, world!" });
+// });
+
+router.get("/energizers", async (req, res) => {
+	try {
+		const result = await db.query("SELECT * FROM energizers");
+		res.json(result.rows);
+	} catch (error) {
+		logger.error(error);
+		res.status(500).json(error);
+	}
+});
```

Comment:
Azin has updated this endpoint in her latest pull/merge. please make a git pull and git merge before we merge your code